packages/xo-web/src/xo-app/xoa/update/index.js

```diff
@@ -51,13 +51,13 @@ const states = {
 }
 
 const update = () => xoaUpdater.update()
-const upgrade = ({ runningJobsExist }) =>
+const _upgrade = ({ upgrade, runningJobsExist }) =>
   runningJobsExist
     ? confirm({
         title: _('upgradeWarningTitle'),
         body: _('upgradeWarningMessage'),
-      }).then(() => xoaUpdater.upgrade())
-    : xoaUpdater.upgrade()
+      }).then(() => xoaUpdater.upgrade(upgrade))
+    : xoaUpdater.upgrade(upgrade)
```

Comment:
So it will actually do the upgrade if there's an active trial? Why would you want this?

Reply:
I use this to have notification `Start updating` instead of `Start upgrading `when we downgrade.